django/contrib/contenttypes/fields.py

```diff
@@ -226,8 +224,8 @@ def __get__(self, instance, cls=None):
         pk_val = getattr(instance, self.fk_field)
 
         try:
-            rel_obj = getattr(instance, self.cache_attr)
-        except AttributeError:
+            rel_obj = self.get_cached_value(instance)
```

Comment:
Adding a default kwarg doesn't mean we have to use it everywhere, and doesn't necessarily mean it defaults to None either. We could default to a sentinel object causing a raise. You would only get a default of None of you explicitly specify one. It's true that would be different from `dict.get`. Still seems potentially worth it to me, but I'm happy to go with your preference.